daemon/daemon_test.go

```diff
@@ -335,6 +337,43 @@ func TestDaemonReloadLabels(t *testing.T) {
 	}
 }
 
+func TestDaemonReloadInsecureRegistries(t *testing.T) {
+	daemon := &Daemon{}
+	daemon.RegistryService = registry.NewService(registry.ServiceOptions{})
+	daemon.configStore = &Config{}
+
+	valuesSets := make(map[string]interface{})
+	valuesSets["insecure-registries"] = "10.10.1.11:5000"
+
+	insecureRegistries := []string{"10.10.1.11:5000", "10.10.1.12:5000"}
+	newConfig := &Config{
+		CommonConfig: CommonConfig{
+			ServiceOptions: registry.ServiceOptions{
+				InsecureRegistries: insecureRegistries,
+			},
+			valuesSet: valuesSets,
+		},
+	}
+
+	daemon.Reload(newConfig)
+
+	registries := daemon.RegistryService.ServiceConfig()
+
+	var data []string
+
+	for _, value := range registries.InsecureRegistryCIDRs {
+		data = append(data, value.IP.String())
+	}
+
+	insecureRegistriesStr := strings.Join(data, " ")
+
+	for _, value := range insecureRegistries {
+		if strings.Contains(insecureRegistriesStr, value) {
```

Comment:
No idea what this test is doing (as a whole), so probably need to refactor, but...
Shouldn't this be `!strings.Contains`?